tools/gen-unicode.py

```diff
@@ -783,29 +677,33 @@ def generate_conversions(script_args):
 
 
 def main():
-    parser = argparse.ArgumentParser(description='lit-unicode-{conversions,ranges}.inc.h generator',
+    parser = argparse.ArgumentParser(description='lit-unicode-{conversions,ranges}-{sup}.inc.h generator',
                                      epilog='''
                                         The input files (UnicodeData.txt, SpecialCasing.txt)
```

Comment:
The epilog also needs an update on which files are used (e.g.: "DerivedCoreProperties")